src/main/resources/templates/db_scripts/schema.json

```diff
@@ -115,6 +115,11 @@
           "tOps": "ADD",
           "caseSensitive": false,
           "removeAccents": true
+        },
+        {
+          "fieldName": "identifiers",
+          "arraySubfield" : "value",
+          "arrayModifiers": ["identifierTypeId"]
         }
```

Comment:
This index appears to have been added to the top-level `identifiers` property of loans. 

My understanding is that this work is intended to facilitate searching requests by isbn.

Please can you help me understand the intent of this index?

Reply:
My apologies-at an early stage of this work, I had this incorrectly positioned in the loans table-I later realized my mistake and put it in the correct place (if you scroll down, you'll see there's identical notation in the request table) I must have forgotten to remove the earlier entry.  I'll fix.